src/components/EntityDesignerApp/EntityConfigMetaView/generateEntities/pipeline-runner/runCreateEntitiesStep.ts

```diff
@@ -162,8 +162,7 @@ exist in the dataset "${sourceDataset.name}". Could not find Dataset Column ID
         datasourceId: datasetId,
         entityFieldConfigId: valueExtractor.entityFieldConfigId,
       };
-    })
-    .exhaustive();
+    });
 }
```

Comment:
@dsound-zz The type error is because you need to add `.exhaustive()` back. It is used to verify that this `match()` is exhaustive (i.e. that it is checking all valid values for `valuePickerRuleType`).

`match()` is just like a `switch` statement, except it is a library that lets you do switch statements in a type safe way. It lets you make sure your cases are exhaustive. That way, if we ever add another value to the enum, it will be checked here. It will throw an error telling us that there is another case that hasn't been checked.